docs/user-guide/quickstarts/generic.md

```diff
@@ -0,0 +1,130 @@
+
+In our example, we state that all our jobs will run in the same Docker container image "buildpack-deps". The `image` is usually defined in the form of "repo_name". Alternatively, you can define the image as "repo_name:tag_label", where "tag_label" is a version. See the [Docker documentation](https://hub.docker.com/r/library/buildpack-deps/) for more information.
+
+```yaml
+# Shared definition block
+shared:
+  image: buildpack-deps
+```
+
+#### Jobs
+The `jobs` section is where all the tasks (or `steps`) that each job will execute is defined. All pipelines have "main" implicitly defined. The definitions in your screwdriver.yaml file will override the implied defaults.
+
+### Steps
+The `steps` section contains a list of commands to execute.
+Each step takes the form "step_name: command_to_run". The "step_name" is a convenient label to reference it by. The
+"command_to_run" is the single command that is executed during this step.
+
+In our example, our "main" job executes a simple piece of inline bash code. We also define another job called "second_job". In this job, we intend on running a different set of commands. The "make_target" step calls a Makefile target to perform some set of actions. This is incredibly useful when you need to perform a multi-line command.
+The "run_arbitrary_script" executes a script. This is an alternative to a Makefile target where you want to run a series of commands related to this step.
+
+```yaml
+# Job definition block
+# "main" is a default job that all pipelines have
+jobs:
+  main:
+    # Steps definition block.
+    steps:
+      - hello: echo hi world
+  second_job:
+    steps:
+      - make_target: make greetings
+      - run_arbitrary_script: ./my_script.sh
+```
+
+Now that we have a working repository, let's head over to the Screwdriver UI to build and deploy an app.
+
+## Building with Screwdriver
+
+In order to use Screwdriver, you will need to login to Screwdriver using Github, set up your pipeline, and start a build.
+
+### Login with Github
+
+1. Navigate to the Screwdriver page at [cd.screwdriver.cd](https://cd.screwdriver.cd).
+
+1. Click Login. Click Login with SCM Provider.
+
+1. You will be asked to give Screwdriver access to your repositories. Choose appropriately and click Authorize.
+
+### Create a New Pipeline
+
+1. Click on the Create icon.
+
+1. Enter your repository link into the field. SSH or HTTPS link is fine, with #{BRANCH_NAME} immediately after (ex: `git@github.com:screwdriver-cd/guide.git#master`).
+Click Use this repository to confirm and then click Create Pipeline.
+
+### 4. Start your First build
+Now that you've created a pipeline, you should be directed to your new pipeline page. Click the Start button to start your build.
+
+
+## Congratulations! You just built and ran your first app using Screwdriver!
+
+## Next Steps
+
+Screwdriver will automatically trigger a build every time you push new code to your repository. Let’s try it out.
```

Comment:
I guess we shouldn't mention the auto-trigger then if we don't automagically add hooks yet